blocks/graph.py

```diff
@@ -354,3 +414,104 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, drop_prob=0.5, rng=None,
```

Comment:
Like I said, I'm actually fine with not giving a default :p I was just playing devil's advocate haha Let's just leave it up to @dmitriy-serdyuk